tests/test_validator_plist.py

```diff
@@ -647,6 +647,19 @@ def test_validators_plist_ufo3_contents_missing_file_fail():
     assert len(ufolib_fail_list) == 0
 
 
+def test_validators_plist_ufo3_contents_missing_file_fail(capsys):
+    contents_ufo_path = os.path.join(contents_test_dir_failpath, 'UFO3-UnlistedGlifs.ufo')
+    contents_validator = plistvalidators.ContentsPlistValidator(contents_ufo_path, 3, ufo3_dir_list)
+
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        contents_validator.run_ufolib_import_validation()
```

Comment:
IIRC, I followed the pattern of naming the behavior as a `*_import_validation` method when the validation relied upon external ufolib validators.  You are adding new functionality and it might be useful to refactor this check into a new method.  I'm fine merging this as is so that the test is available and we can address it down the road.